test/suites/drift/suite_test.go

```diff
@@ -269,4 +271,116 @@ var _ = Describe("Drift", Label("AWS"), func() {
 		env.ExpectUpdated(pod)
 		env.EventuallyExpectNotFound(pod, node)
 	})
+	Context("Provisioner Drift", func() {
+		var pod *v1.Pod
+		var provider *v1alpha1.AWSNodeTemplate
```

Comment:
nit: I think we should start calling this `nodeTemplate` in our testing since it's no longer the provider field anymore